src/components/formElements/FormElements.tsx

```diff
@@ -27,7 +28,11 @@ interface BooleanRadioGroupProps extends Omit<RadioGroupProps, 'value' | 'onChan
 export const BooleanRadioGroup = ({ ref, labels, value, onChange, ...props }: BooleanRadioGroupProps) => {
     const { formatMessage } = useI18n({ messages: nb });
     return (
-        <RadioGroup {...props} value={value?.toString()} onChange={(val) => onChange(val === true.toString())}>
+        <RadioGroup
+            {...props}
+            defaultValue={value?.toString() ?? undefined}
+            onChange={(val) => onChange(val === true.toString())}
```

Comment:
Jeg ser dette er gjort en god del ganger (bytte til `defaultValue`), men det blir litt feil. Vi ønsker her kontrollerte inputs, og da må vi bruke `value` i kombinasjon med `onChange`. Tipper at feilene du så i konsollen var om endring fra ukontrollert til kontrollert input? I så fall tror jeg en riktigere fiks vil være

```suggestion
            value={value?.toString() ?? ''}
            onChange={(val) => onChange(val === true.toString())}
```

og da tilsvarende de andre plassene også.

Reply:
Er det en grunn til at vi _ikke_ vil ha uncontrolled inputs?

Reply:
Tror vi egentlig kunne ha brukt det flere plasser enn vi gjør i dag, men man må da ha tunga litt rett i munnen.
Akkurat i dette tilfellet så ville da i så fall `value` blitt til en `string`, og vi hadde blitt nødt til å håndtere det en annen plass.

Reply:
Men value er en sting i input uansett? Godt mulig controlled er det vi burde gå for, bare kjenner ikke til best practice/argumentasjonen for det ene eller andre 😄

Reply:
Ah, ja, ,men det vi gjør her er å kontrollere inputen sånn at vi sender en `boolean` til `onChange`, sånn at skjematilstanden forholder seg til en `boolean`.

Reply:
36c2c9eb4592612e45e04ef61d3052069e5a3a8c